Makefile

```diff
@@ -18,6 +18,12 @@ ifeq ($(BUILD_TYPE),release)
 else ifeq ($(BUILD_TYPE),debug)
 	PG_CONFIGURE_OPTS = --enable-debug --with-openssl --enable-cassert --enable-depend
 	PG_CFLAGS = -O0 -g3 $(CFLAGS)
+else ifeq ($(BUILD_TYPE),release-with-sanitizers)
```

Comment:
I've just realised that some tests may rely on the `BUILD_TYPE` value, i.e., they expect it to be `release` or `debug`.

The thing I wanted by suggesting it:
- support in Makefile
- a way to distinguish tests with enabled sanitisers in a database with test results (as far as I can see in the database they're still stored as `release`). We can add an additional bool column for sanitizers instead

Reply:
Yeah, I see. Then please look at another implementation, with sanitizers activated by
`test-cfg: '[{"pg_version":"v17", "sanitizers": "enabled"}]'`